test/FormGroupSpec.js

```diff
@@ -46,6 +46,15 @@ describe('FormGroup', function() {
     assert.ok(ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'form-group-lg'));
   });
 
+  // This test case must come first, since the error only gets logged once.
```

Comment:
It is just my curiosity.
I had placed this test after next one and then placed it into the end of file
and in both cases all tests are green.

What did you mean by this comment ?